content/concept-partial-payments.md

```diff
@@ -0,0 +1,90 @@
+
+Partial payments can be used to exploit naive integrations with the Ripple Consensus Ledger to steal money from exchanges and gateways. The [Partial Payments Exploit](#partial-payments-exploit) section of this document describes how this exploit works and how you can avoid it.
+
+## Semantics
+
+### Without Partial Payments
+
+When sending a Payment that does not use the Partial Payment flag, the `Amount` field of the transaction specifies the exact amount to deliver, and the `SendMax` field specifies the maximum amount and currency to send. If payment cannot deliver the full `Amount` without exceeding the `SendMax` parameter, or the full amount cannot be delivered for any other reason, the transaction fails. If the `SendMax` field is omitted from the transaction instructions, it is considered to be equal to the `Amount`. In this case, the payment can only succeed if the total amount of fees is 0.
+
+In other words:
+
+    Amount + (fees) = (sent amount) ≤ SendMax
+
+In this formula, "fees" refers to [transfer fees](concept-transfer-fees.html) and currency exchange rates. The "sent amount" and the delivered amount (`Amount`) may be denominated in different currencies and converted by consuming Offers in the Ripple Consensus Ledger's decentralized exchange.
+
+**Note:** The `Fee` field of the transaction refers to the XRP [transaction cost](concept-transaction-cost.html), which is destroyed to relay the transaction to the network. The exact transaction cost specified is always debited from the sender and is completely separate from the fee calculations for any type of payment.
+
+### With Partial Payments
+
+When sending a Payment that has the Partial Payment flag enabled, the `Amount` field of the transaction specifies a maximum amount to deliver. Partial payments can succeed at sending _some_ of the intended value despite limitations including fees, lack of liquidity, insufficient space in the receiving account's trust lines, or other reasons.
+
+The optional `DeliverMin` field specifies a minimum amount to deliver. The `SendMax` field functions the same as with non-partial payments. The partial payment transaction is successful if it delivers any amount equal or greater than the `DeliverMin` field without exceeding the `SendMax` amount. If the `DeliverMin` field is not specified, a partial payment can succeed by delivering any positive amount.
+
+In other words:
+
+    Amount ≥ (Delivered Amount) = SendMax - (Fees) ≥ DeliverMin > 0
+
+### Partial Payment Limitations
+
+Partial Payments have the following limitations:
+
+- A partial payment cannot provide the XRP to fund an address; this case returns the [result code][] `telNO_DST_PARTIAL`.
+- Direct XRP-to-XRP payments cannot be partial payments; this case returns the [result code][] `temBAD_SEND_XRP_PARTIAL`.
+    - However, issuance-to-XRP payments or XRP-to-issuance payments _can_ be partial payments.
+
+[result code]: reference-transaction-format.html#transaction-results
```

Comment:
This reference defines the link used in lines 37 and 38; line 41 does not display in the parsed markdown. Search for _implicit link name_ in [the original Markdown definition](https://daringfireball.net/projects/markdown/syntax)